src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -832,27 +831,26 @@ private MenuBar createMenu() {
         }
 
         edit.getItems().addAll(
-                               factory.createMenuItem(StandardActions.MANAGE_KEYWORDS, new ManageKeywordsAction(this)),
-                               factory.createMenuItem(StandardActions.REPLACE_ALL, new OldDatabaseCommandWrapper(Actions.REPLACE_ALL, this, Globals.stateManager)),
-                               factory.createMenuItem(StandardActions.MASS_SET_FIELDS, new MassSetFieldAction(this))
+                factory.createMenuItem(StandardActions.MANAGE_KEYWORDS, new ManageKeywordsAction(this)),
```

Comment:
This will create again merge conflicts with the indentation

Reply:
Yes, I think, we need to revisist the eclipse formatting rules. Having such a huge indention for method arguments is really ugly.